wdl/tasks/QC/AlignedMetrics.wdl

```diff
@@ -24,6 +24,10 @@ workflow AlignedMetrics {
         File ref_dict
 
         String? gcs_output_dir
+        
+        Int? mem_gb_input_ReadMetrics  # Optional input for memory in GiB
```

Comment:
Actually, it might be better to define the inputs as the struct for runtime attributes.  Then you can just bubble that whole struct down.




```
RuntimeAttr? runtime_attr_override
```